### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jtsaito**

### CODE REVIEW

**Review by awendt — APPROVED** [review]
🥳

File: README.md [hunk 1]
```diff
@@ -3,8 +3,6 @@
 This organization is intended to host (private) source code repositories for **Babbel Hackday projects and alike**.
 It is owned and maintained by the Babbel Infrastructure Team.
 
-Please reach out to Infrastructure Team via #ask-infrastructure Channel on Slack if
+If you need access to this organizations and already are represented in [users.infrastructure](https://github.com/lessonnine/users.infrastructure), please create a PR to make yourself a member of the `babbel-hackdays` organization.
 
-- you or a member of your team need access to the organization
-- you need help with the configuration of a repository
-- you run in trouble and need any other kind of support
+If you want to organize an event, pleaes collect the list of all non-engineering GitHub users who need access to the organization during your event and contact the Infrastructure Team via #ask-infrastructure Channel on Slack.
```
Comment (jansiwy) [inline README.md:8]:
```suggestion
If you want to organize an event, please collect the list of all non-engineering GitHub users who need access to the organization during your event and contact the Infrastructure Team via #ask-infrastructure Channel on Slack.
```

File: README.md [hunk 2]
```diff
@@ -3,8 +3,6 @@
 This organization is intended to host (private) source code repositories for **Babbel Hackday projects and alike**.
 It is owned and maintained by the Babbel Infrastructure Team.
 
-Please reach out to Infrastructure Team via #ask-infrastructure Channel on Slack if
+If you need access to this organizations and already are represented in [users.infrastructure](https://github.com/lessonnine/users.infrastructure), please create a PR to make yourself a member of the `babbel-hackdays` organization.
```
Comment (jansiwy) [inline README.md:6]: I'd recommend to extend https://doc.babbel.com/lessonnine/guides.doc/tooling/github.html with `babbel-hackdays` and then link that documentation.

**Review by ghost — APPROVED** [review]

**Comment by jtsaito** [comment]
Replaced by: https://github.com/babbel-hackdays/readme/pull/2

**Pull request closed without merging**